extensions/ARM/SPV_ARM_graph.asciidoc

```diff
@@ -0,0 +1,345 @@
+SPV_ARM_graph
+=============
+
+Name Strings
+------------
+
+SPV_ARM_graph
+
+Contact
+-------
+
+To report problems with this extension, please open a new issue at:
+
+https://github.com/KhronosGroup/SPIRV-Headers
+
+Contributors
+------------
+
+- Kevin Petit, Arm Ltd. +
+
+Notice
+------
+
+Copyright (c) 2022-2025 Arm Ltd.
+
+Status
+------
+
+Complete.
+
+Version
+-------
+
+[width="40%",cols="25,25"]
+|========================================
+| Last Modified Date | 2025-06-11
+| Revision           | 1
+|========================================
+
+Dependencies
+------------
+
+This extension is written against the SPIR-V Specification,
+Version 1.6 Revision 2.
+
+This extension requires SPIR-V 1.6.
+
+This extension requires SPV_ARM_tensors.
+
+Overview
+--------
+
+This extension adds support for graphs to SPIR-V. Graphs provide a
+construct to express dataflow computations operating on full resources (only
+tensors are supported with this extension but graphs could also be used with
+images). This extension does not define any operations for use within graphs.
+
+Extension Name
+--------------
+
+To use this extension within a SPIR-V module, the following
+*OpExtension* must be present in the module:
+
+----
+OpExtension "SPV_ARM_graph"
+----
+
+Modifications to the SPIR-V Specification, Version 1.6
+------------------------------------------------------
+
+Terms
+~~~~~
+
+'Graph': A dataflow graph defined by an *OpGraphARM*. A graph may consume inputs
+and produces at least one output.
+
+'Graph entry point': An *OpGraphARM* exposed to the execution environment using
+a *OpGraphEntryPointARM* instruction.
+
+'Graph Interface Type': An *OpTypeTensorARM*, *OpTypeArray* of *OpTypeTensorARM*, or
+*OpTypeRuntimeArray* of *OpTypeTensorARM*.
+
+Logical Layout of a Module
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This extension changes the logical layout of modules as follows:
+
+- Add a new section after 11 Function definitions where graph definitions and entry points reside.
```

Comment:
Didn't you mean to define graph definition here ? there is no defs attached here (only the last 2 points of the validation rules)

Reply:
Yes, better to expand on it here as is done for functions in the core spec. Thanks!